tools/topology/platform/intel/adl.m4

```diff
@@ -0,0 +1,38 @@
+#
+# Tigerlake differentiation for pipelines and components
```

Comment:
so will change Tigerlake to Alderlake later?

Reply:
I have to figure out if this file is actually necessary. No one seems to have missed it so far :-)